doc/sources/.static/dark.css

```diff
@@ -0,0 +1,514 @@
+    border-color: currentcolor currentcolor currentcolor rgb(23, 82, 153);
+    color: rgb(207, 203, 196);
+}
+.footerlinks table {
+    background-color: rgb(32, 35, 37);
+}
+#content pre {
+    background-color: rgb(32, 35, 37);
+}
+#content h2 {
+    border-bottom-color: rgb(57, 61, 64);
+}
+#content div.warning {
+    border-left-color: rgb(124, 23, 23);
+}
+#content div.warning p.first {
+    color: rgb(231, 123, 123);
+}
+#content div.note {
+    border-left-color: rgb(47, 109, 47);
+}
+#content div.note p.first {
+    color: rgb(131, 201, 131);
+}
+.versionchanged,
+.versionadded {
+    color: rgb(172, 165, 154);
+}
+.bodyshortcut {
+    background-color: rgb(32, 35, 37);
+}
+dl.api-level dt {
+    background-color: rgb(42, 45, 47);
+}
+dl.api-level dl.attribute dt {
+    background-color: rgb(32, 35, 37);
+}
+tt.descname {
+    color: rgb(181, 174, 165);
+}
+em.property {
+    color: rgb(226, 126, 64);
+}
+table.field-list {
+    border-color: currentcolor !important;
+}
+table.field-list th.field-name {
+    background-color: rgb(32, 35, 37);
+}
+table.field-list tr {
+    border-color: rgb(48, 52, 54);
+}
+table.field-list td.field-body dl.docutils dt {
+    background-color: rgb(32, 35, 37);
+}
+form.search input {
+    border-color: rgb(61, 66, 69);
+}
+table.docutils {
+    border-color: rgb(61, 66, 69);
+}
+.toc {
+    background-color: rgb(30, 32, 33);
+    border-color: rgb(57, 61, 64);
+}
+.toc ul li li li {
+    border-left-color: rgb(57, 61, 64);
+}
+span.versionadded span {
+    color: rgb(62, 149, 226);
+}
+#version_selector {
+    border-color: rgb(61, 66, 69);
+    background-color: rgba(24, 26, 27, 0);
+    background-image: url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMzQiIGhlaWdodD0iMzQiPjxkZWZzPjxmaWx0ZXIgaWQ9ImRhcmtyZWFkZXItaW1hZ2UtZmlsdGVyIj48ZmVDb2xvck1hdHJpeCB0eXBlPSJtYXRyaXgiIHZhbHVlcz0iMC4yNDkgLTAuNjE0IC0wLjY3MiAwLjAwMCAxLjAzNSAtMC42NDYgMC4yODggLTAuNjY0IDAuMDAwIDEuMDIwIC0wLjYzNiAtMC42MDkgMC4yNTAgMC4wMDAgMC45OTQgMC4wMDAgMC4wMDAgMC4wMDAgMS4wMDAgMC4wMDAiIC8+PC9maWx0ZXI+PC9kZWZzPjxpbWFnZSB3aWR0aD0iMzQiIGhlaWdodD0iMzQiIGZpbHRlcj0idXJsKCNkYXJrcmVhZGVyLWltYWdlLWZpbHRlcikiIHhsaW5rOmhyZWY9ImRhdGE6aW1hZ2UvcG5nO2Jhc2U2NCxpVkJPUncwS0dnb0FBQUFOU1VoRVVnQUFBQ0lBQUFBaUNBWUFBQUE2Und2Q0FBQUFBWE5TUjBJQXJzNGM2UUFBQUFaaVMwZEVBUDhBL3dEL29MMm5rd0FBQUFsd1NGbHpBQUFMRXdBQUN4TUJBSnFjR0FBQUFBZDBTVTFGQjl3RUdoSWJNMnBJdWwwQUFBQnVTVVJCVkZqRDdkRkJDc0l3QUFUQUVYMVAyNVBtRjc3Ym52UVpmVU1Wdk9SUWloZGpRY1FkQ0FsWkVwYUVpSWlJK0ZQN2hqTTl6cGd4cmJLdVp2Y1gyZVpGSmp4d1dwWHBVSERCN2QxTEQ0MHZlYTF6V2V3VmpDMGxZUGZoMXc0NDF2VzRLUGdWUXgwUkVSSHhVNTV0cmhBTTY0V0JzUUFBQUFCSlJVNUVya0pnZ2c9PSIgLz48L3N2Zz4=");
+}
+code.docutils {
+    background-color: rgb(36, 39, 41);
+    background-image: none;
+    color: rgb(232, 230, 227);
+}
+@media (min-width: 1024px) {
+    .toc {
+        background-color: rgb(30, 32, 33);
+        border-color: rgb(57, 61, 64);
+    }
+}
```

Comment:
This one shouldn't be needed cause the one without the media query gets applied also on `width>=1024px`.